src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1368,7 +1368,7 @@ MulticopterAttitudeControl::start()
 	_control_task = px4_task_spawn_cmd("mc_att_control",
 					   SCHED_DEFAULT,
 					   SCHED_PRIORITY_MAX - 5,
-					   1700,
+					   2200,
```

Comment:
Can you investigate why it's that much higher? And maybe ifdef if we can't get it reduced?